manifests/zfs-operator.yaml

```diff
@@ -1884,6 +1971,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["storageclasses", "csinodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: [ "storage.k8s.io" ]
```

Comment:
Why spaces here?

Reply:
I have copied it as it is from upstream, i can fix it but probably would be nice if i made a PR upstream as we are going to diverge otherwise